.circleci/config.yml

```diff
@@ -278,107 +296,6 @@ commands:
             source $BASH_ENV
 
 
-  run-linter-check:
-    description: "Run linter check"
-    steps:
-      - run:
-          name: Linter check
-          command: |
-            ng lint ddp-sdk
-            ng lint toolkit
-            ng lint $ANGULAR_PROJECT_NAME
-  run-tests:
-    description: "Run all tests"
-    steps:
-      - run:
-          name: ng test
-          command: |
-            mkdir -p /tmp/junit
-            ng test ddp-sdk --watch=false --browsers ChromeHeadlessNoSandbox --reporters junit
-            ng test toolkit --watch=false --browsers ChromeHeadlessNoSandbox --reporters junit
-            ng test $ANGULAR_PROJECT_NAME --watch=false --browsers ChromeHeadlessNoSandbox --reporters junit
-          environment:
-            JUNIT_REPORT_PATH: /tmp/junit/
-            JUNIT_REPORT_NAME: test-results.xml
-          when: always
-      - store_test_results:
-          path: /tmp/junit
-      - store_artifacts:
-          path: /tmp/junit
-
-  build-atcp-assets:
```

Comment:
`build-atcp-assets` and `execute-study-specific-commands` are deleted. Steps were combined into `build` command.